config/config.exs

```diff
@@ -28,16 +28,16 @@ config :snapeth, Snapeth.Scheduler,
   jobs: [
     # Every Monday at 2:00
     {"0 14 * * MON",      {Snapeth, :display_leaderboard, []}},
-    # Every Monday at 2:01
-    {"1 14 * * MON",      {Snapeth, :clear_leaderboard, []}},
+    # Every year
+    {"0 0 1 1 *",         {Snapeth, :clear_leaderboard, []}},
```

Comment:
the clear_leaderboard function in storage.ex needs to be adapted to wipe all leaderboards from S3